### DISCUSSION TITLE
Continuously marked watched at trakt

### DISCUSSION THREAD

**Discussion by jackfowl** [Taxel/PlexTraktSync#701] [Q&A] [answered]
One episode is constantly marked as watched on trakt every time I run the script. How can I avoid that behaviour? I opened a issue report, but as this become non reproducible it was closed. If someone has the same issue or solved that problem could give a helpful hand?

**Comment by stevietv** [comment]
what @glensc asked in that issue was that you use the bug report issue template. Its not investigable in the way you posted it in #697

**Reply by glensc** [reply, marked as answer]
Exactly. I have no time and wish to deal with bug reports where reporter doesn't even bother to fill details of their installation.
